.travis.yml

```diff
@@ -26,3 +26,6 @@ before_script:
 script:
 - make verify-codegen
 - make test-all
+
+after_success:
+- .ci/build-master.sh
```

Comment:
We could move the conditional from bash to the `.travis.yml`

https://docs.travis-ci.com/user/conditional-builds-stages-jobs/

Reply:
I tried but gave up reading documentation and try a few combinations.